deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -222,8 +225,56 @@ def run_on_pred(self, y_true, y_pred=None, y_proba=None):
             return self.scorer._score_func(y_true, pred_to_use, **self.scorer._kwargs) * self.scorer._sign
         raise errors.DeepchecksValueError('Only supports sklearn scorers')
 
+    def _wrap_classification_model(self, model, label_col: pd.Series):
+        """Convert labels to 0/1 if model is a binary classifier, update classes_ property and pad probas."""
+
+        class MyModelWrapper:
+            """Convert labels to 0/1 if model is a binary classifier, update classes_ property and pad probas."""
+
+            def __init__(self, user_model, possible_classes):
+                self.user_model = user_model
+                self.possible_classes = possible_classes
+                if hasattr(model, 'classes_') and len(model.classes_) > 0 \
+                        and len(model.classes_) != len(possible_classes):
+                    self.indexes_to_pad_around = [possible_classes.index(x) for x in list(model.classes_)]
```

Comment:
That basically covers new labels in test that where not present in training, right?

Reply:
yep